lib/rubocop/sketchup/cop/requirements/init_entity.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module SketchupRequirements
+      # SketchUp Entity objects (`Face`, `Edge`, `Group` etc) should not be
+      # initialized using the `new` constructor. These objects are managed by
+      # SketchUp. Instead use the `Sketchup::Entities#add_*` methods to create
+      # them.
+      class InitEntity < SketchUp::Cop
+
+        # TODO: Check more classes.
+        def_node_matcher :init_entity?, <<-PATTERN
+          (send
+            (const
+              (const nil? :Sketchup)
+              {:Face :Group :Edge :ConmponentInsstance :Image}
+            )
+            :new ...
+          )
+        PATTERN
+
+        MSG_SET_NAME = 'Entity objects (Face, Edge, Group etc) are managed ' \
```

Comment:
"MSG_SET_NAME"? Not sure what "set name" refers to in this context.

Reply:
Copypasta... Fixed now!